compiler/noirc_frontend/src/hir/type_check/errors.rs

```diff
@@ -99,6 +99,8 @@ pub enum TypeCheckError {
     CannotMutateImmutableVariable { name: String, span: Span },
     #[error("No method named '{method_name}' found for type '{object_type}'")]
     UnresolvedMethodCall { method_name: String, object_type: Type, span: Span },
+    #[error("No method named '{method_name}' found for type '{object_type}'")]
```

Comment:
```suggestion
    #[error("To invoke function field '{method_name}' on type '{object_type}' it must be surrounded with parentheses")]
```

Reply:
I see the diagnostic below also starts with "no method", but here it looks like a copy of the message from `UnresolvedMethodCall`; I thought some distinction would still be helpful.